### PR TITLE
Re-Revert the use of utils.pm in main.pm

### PR DESCRIPTION
**Pull request by coolo**
We need to massage the search path before we can use utils.pm

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: products/sle/main.pm [hunk 1]
```diff
@@ -5,6 +5,13 @@ use lockapi;
 use autotest;
 use needle;
 use File::Find;
+use File::Basename;
+
+BEGIN {
+    # add search path for lib/utils.pm
+    unshift @INC, dirname(__FILE__) . '/../../lib';
+}
+use utils;
```
Comment (aaannz) [inline products/sle/main.pm]: I'm for to do this in isotovideo as we do for test modules in autotest.pm.

Comment (coolo) [inline products/sle/main.pm]: yes, but that would require a new deployment. We can do it in both places for the time being and remove later

**Comment by coolo** [comment]
Tracked the follow up as https://progress.opensuse.org/issues/12258

**Pull request merged**